build.gradle

```diff
@@ -51,17 +51,12 @@ subprojects {
     apply plugin: "jacoco-report-aggregation"
     apply plugin: "groovy"
 
-    tasks.withType(JavaCompile) {
+    tasks.withType(JavaCompile).configureEach {
         options.compilerArgs << "-Xlint:unchecked"
         options.compilerArgs << "-Xlint:deprecation"
-    }
 
-    project(":polaris-service") {
-        apply plugin: "application"
```

Comment:
For my own reference, Plugin specification is moved into project specific build files.